src/Data/Repositories/NodeRepository.cs

```diff
@@ -109,19 +110,21 @@ public async Task<List<Node>> GetAll()
                 .ToListAsync();
         }
 
-        public async Task<List<Node>> GetAllManagedByNodeGuard()
+        public async Task<List<Node>> GetAllManagedByNodeGuard(bool includeDisabledNodes = true)
         {
             await using var applicationDbContext = await _dbContextFactory.CreateDbContextAsync();
 
-            var resultAsync = await applicationDbContext.Nodes
+            var query = applicationDbContext.Nodes
                 .Include(x => x.ReturningFundsWallet)
                 .ThenInclude(x => x.Keys)
-                .Where(node => node.Endpoint != null)
-                .ToListAsync();
+                .Include(x => x.ReturningFundsWallet)
+                .Where(node => node.Endpoint != null);
+
```

Comment:
Is it missing the conditional for including/excluding the disabled nodes?

Reply:
Now now sense to add this the logic because if node disabled anyway it should be visible.

Reply:
But then why having the parameter in the function?

Reply:
The parameter is not needed anymore. Enable/disabled nodes should be always visible and you can see the status in a new column. Soft deleted nodes are the ones that shouldn't be visible and that is taken care of by deleting the endpoint and macaroon